apps/browser-extension-wallet/src/providers/AnalyticsProvider/analyticsTracker/AnalyticsTracker.ts

```diff
@@ -60,31 +60,46 @@ export class AnalyticsTracker {
     }
   }
 
+  private async checkNewSessionStarted(): Promise<void> {
+    if (!this.postHogClient) {
+      console.debug('[ANALYTICS] no posthog client');
+      return;
+    }
+    if (await this.userIdService.isNewSession()) {
+      await this.postHogClient?.sendSessionStartEvent();
```

Comment:
```suggestion
      await this.postHogClient.sendSessionStartEvent();
```
(you're asserting above that PostHog client is defined)